File: Solutions/001.二维数组中的查找/Solution1.cpp
//运行时间（leetcode240）：1248ms
class Solution {
public:
    bool Find(int target, vector<vector<int> > array) {
        for (auto vec: array)
            for (auto v: vec)
                if (v == target)
                    return true ;
        return false ;
    }
};

File: Solutions/001.二维数组中的查找/Solution1.java
public class Solution {
    public boolean Find(int target, int [][] array) {
        for (int i = 0; i < array.length; i++) {
            for (int j = 0; j < array[i].length; j++) {
                if (target == array[i][j]) {
                    return true;
                }
                if (target < array[i][j]) {
                    break;
                }
            }
        }
        return false;
    }
}